### PR TITLE
Add troubleshooting cheatsheet

### PR DESCRIPTION
**Pull request by dholbach**
Thanks @pjbgf for providing content

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-1033--fluxcd.netlify.app/docs/cheatsheets/troubleshooting/

File: content/en/docs/cheatsheets/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+```cli
+kubectl get kustomizations.kustomize.toolkit.fluxcd.io -A
+kubectl get helmreleases.helm.toolkit.fluxcd.io -A
+kubectl get helmcharts.source.toolkit.fluxcd.io -A
+```
+
+Looking for controller errors:
+
+```cli
+kubectl logs -n flux-system -l app=source-controller | grep error
```
Comment (stefanprodan) [inline content/en/docs/cheatsheets/troubleshooting.md:58]: Please replace this with an example of `flux logs`

**Comment by stefanprodan** [comment]
I think all the debugging things in FAQ should move here, and have a generic Q in the FAQ to point here.

File: content/en/docs/cheatsheets/troubleshooting.md (hunk 1 above)
Comment (pjbgf) [reply content/en/docs/cheatsheets/troubleshooting.md:58]: yep, let's replace it with:
`flux logs --all-namespaces --level=error`

**Pull request merged**